myhpi/tenca_django/templates/tenca_django/dashboard.html

```diff
@@ -0,0 +1,48 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load bootstrap4 %}
+{% load tenca_extras %}
+
+{% block title %}
+	{% trans "List memberships" %}
+{% endblock %}
+
+{% block breadcrumbs %}
+	<li><span class="fa fa-angle-right" aria-hidden="true"></span>&nbsp;{% trans "Mailing lists" %}</li>
+{% endblock %}
+
+{% block content %}
+	<h4>{% trans "Create new list" %}</h4>
+	<form method="post">
+		{% csrf_token %}
+		{% bootstrap_field form.list_name show_label=False addon_after=domain_addon %}
+		{% buttons %}
+			<button type="submit" class="btn btn-primary">{% trans "Create" %}</button>
+		{% endbuttons %}
+	</form>
+
+	{% if memberships %}
+		<h4 class="title is-4">{% trans "Your Memberships" %}</h4>
+
+		{% for list_id, hash_id, is_owner in memberships %}
+			<div class="card mb-1">
```

Comment:
this was just the ugly design I built back then, maybe @SilvanVerhoeven wants to come up with something more fancy

Reply:
moved to #161